testing/features/step_definitions/navigation_steps.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+Given("the navigation is on the page") do
+  @component = Navigation::Standard.new.tap(&:load)
```

Comment:
General question more for @luke-hill here - how come this is the conventional way to call `#load` on these classes? Is there scope for making it either one of these three things:

1. Happen automatically as part of `.new` (I'm guessing there are good reasons not to do this one!)
2. Happen if a given parameter is passed to `.new` (e.g. `Navigation::Standard.new(load: true)`)
3. Or at least have the `#load` method return `self` so that you can call it as `.new.load` instead of `.new.tap(&:load)`?

Reply:
So.

`#load` as a method will do about 4 things under the hood: https://github.com/site-prism/site_prism/blob/main/lib/site_prism/page.rb#L39-L57

When you instantiate a new page class, it just returns an OO object. So you can't do anything on instantiation (Yet, see below).
When instantiating a page object. We don't like users passing in much (As a few things need to be passed in).
The load method "could" return `self`, but then that conflicts with the specification, which says it returns a yielded object (This is to support Capybara simple / string objects).
By default most of the time load will return `true`. This is to stipulate that it has passed the requisite load validation tests.

However, that all being said, a long standing branch of mine: https://github.com/site-prism/site_prism/tree/feature/auto_run_validations would allow load validations to be automatically ran when calling `#new` - But I've not got time / priority to work on this. This is something that will likely land in 4.x at some point.

Reply:
@luke-hill This is very clear. Thank you for the explanation!